ci-operator/step-registry/ipi/conf/alibabacloud/ipi-conf-alibabacloud-ref.yaml

```diff
@@ -0,0 +1,40 @@
+ref:
+  as: ipi-conf-alibabacloud
+  from: tools
+  commands: ipi-conf-alibabacloud-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+  - name: SIZE_VARIANT
```

Comment:
nit: you should either grow the script to respect this and other variables, or drop them from the ref docs.

Reply:
I originally had a much more robust script following the AWS example. After much headache trying to iron out a bunch of small bugs I decided to remove 90% of the code and take a much simpler approach. This is an artifact from a previous removal. I'll clean it up.